spec/features/newflow/user_claims_account_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+feature 'User claims an unclaimed account' do
+  before do
+    turn_on_feature_flag
+  end
+
+  background { load 'db/seeds.rb' }
+  let!(:app)   { create_default_application }
+  let(:user_email) { 'unclaimeduser@example.com' }
+  let(:user_options) {
+    {
+      email: user_email,
+      application: app,
+      username: 'therulerofallthings',
+      first_name: Faker::Name.first_name,
+      last_name: Faker::Name.last_name,
+      already_verified: false
+    }
+  }
+
+  def visit_invite_url
+    delivery = ActionMailer::Base.deliveries.last
+    match = delivery.body.encoded.match(/(confirm\/unclaimed\?code=\w+)/)
+    expect(match).to_not be_nil
+    visit '/' + match.captures.first
```

Comment:
No idea why only one of the tests fails when I don't prepend `'/'`. I don't know what is rewriting the url that prepends `/i` if the path doesn't start with `/`.